### PR TITLE
Update translation catalogs

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Update translation catalog
| Type?         | improvement
| Category?     | LO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | ~
| How to test?  | ~

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/21607)

### CODE REVIEW

**Review by Progi1984 — DISMISSED** [review]

**Review by matthieu-rolland — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

**Review by matks — DISMISSED** [review]

**Review by sowbiba — DISMISSED** [review]

**Review by matthieu-rolland — DISMISSED** [review]

**Review by Progi1984 — DISMISSED** [review]

File: .github/workflows/php.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ jobs:
         with:
           php-version: '7.3'
           extensions: mbstring, intl, gd, xml, dom, json, fileinfo, curl, zip, iconv
+          tools: composer:v1
```
Comment (atomiix) [inline .github/workflows/php.yml]: Can't we get rid of that now?

Comment (eternoendless) [reply .github/workflows/php.yml]: +1
```suggestion
```

Comment (jolelievre) [reply .github/workflows/php.yml]: Sorry I forgot to clean this commit! It's ok now

**Review by atomiix — APPROVED** [review]

**Review by matks — APPROVED** [review]

File: app/Resources/translations/default/ModulesBlockreassuranceAdmin.xlf [hunk 2]
```diff
@@ -274,22 +284,22 @@ Comment: Confirm uninstall</note>
...
+        <note>Line: 83</note>
       </trans-unit>
       <trans-unit id="43c7582c40594c3fe32a6634ef9dcb07">
         <source>Deactivated</source>
         <target>Deactivated</target>
-        <note>Line: 85</note>
+        <note>Line: 84</note>
       </trans-unit>
       <trans-unit id="4aa4ae23dfbd6b0941b3749079107b48">
         <source>We recommend 3 blocks at maximum.</source>
```
Comment (Julievrz) [inline app/Resources/translations/default/ModulesBlockreassuranceAdmin.xlf:300]: This sentence sounds very French to me, I don't think we would say "at maximum" in English. I suggest saying " We recommend a maximum of 3 blocks." @LouiseBonnard do you agree?

File: app/Resources/translations/default/ModulesWelcomeAdmin.xlf [hunk 3]
```diff
@@ -5,137 +5,137 @@
...
+        <note>Line: 105</note>
       </trans-unit>
       <trans-unit id="69168bfbea070ea2d4d311cef7c4c39a">
         <source>How much do you want to sell it for?</source>
         <target>How much do you want to sell it for?</target>
-        <note>Line: 119</note>
+        <note>Line: 112</note>
       </trans-unit>
       <trans-unit id="331ca7db26429b18d4fc27a4c0682aa2">
         <source>Yay! You just created your first product. Looks good, right?</source>
```
Comment (Julievrz) [inline app/Resources/translations/default/ModulesWelcomeAdmin.xlf:41]: I don't think this sentence is gramatically correct, we should say "Yay! You *have* just created your first product. Looks good, right?"

Comment (Julievrz) [inline app/Resources/translations/default/ModulesWelcomeAdmin.xlf:41]: I don't think this sentence is gramatically correct, we should say "Yay! You have just created your first product. Looks good, right?"

File: app/Resources/translations/default/ModulesBlockreassuranceAdmin.xlf (hunk 2 above)
Comment (LouiseBonnard) [reply app/Resources/translations/default/ModulesBlockreassuranceAdmin.xlf:300]: PR has already been merged, @Julievrz! But do not hesitate to take note of the improvements you see to suggest them for 1.7.8. :-)

Comment (Julievrz) [reply app/Resources/translations/default/ModulesBlockreassuranceAdmin.xlf:300]: Oops, I didn't see. Ok :)

**Pull request merged**